blockly-core/core/ui/block_space/flyout.js

```diff
@@ -406,8 +406,10 @@ Blockly.Flyout.prototype.show = function(xmlList) {
     y: margin
   };
 
-  // Bind mousedown on the flyout background
-  this.blockSpace_.bindBeginPanDragHandler(this.svgGroup_);
+  // We bind mouse pan-drag on the background group so blocks are ignored.
+  this.blockSpace_.bindBeginPanDragHandler(this.svgBackground_);
+  // We bind scrollwheel on the full group so wheeling over blocks works.
+  this.blockSpace_.bindScrollOnWheelHandler(this.svgGroup_);
```

Comment:
Nice work tracking this down.  Now that I see it, it feels really good to see these two responsibilities broken out.